tests/resources/aria-at-harness.mjs

```diff
@@ -176,266 +287,1179 @@ export function verifyATBehavior(atBehavior) {
+ * @property {InstructionDocumentInstructionsHeader} header
+ * @property {Description} description
+ * @property {InstructionDocumentInstructionsInstructions} instructions
+ * @property {InstructionDocumentInstructionsAssertions} assertions
+ * @property {InstructionDocumentButton} openTestPage
+ */
+
+/**
+ * @typedef InstructionDocument
+ * @property {InstructionDocumentErrors} errors
+ * @property {InstructionDocumentInstructions} instructions
+ * @property {InstructionDocumentResults} results
+ * @property {InstructionDocumentButton} submit
+ */
+
+/**
+ * @param {Behavior} testJson
+ * @param {Commands} commandJson
+ * @param {ResultState} resultState
+ * @param {(action: (state: ResultState) => ResultState) => void} dispatch
+ * @returns {InstructionDocument}
+ */
+export function instructionDocument(testJson, commandJson, resultState, dispatch) {
+  const mode = testJson.mode;
+  const modeInstructions = commapi.getModeInstructions(mode, at);
+  const userInstructions = testJson.specific_user_instruction.split("|");
+  const lastInstruction = userInstructions[userInstructions.length - 1];
+  const setupScriptDescription = testJson.setup_script_description
+    ? ` and runs a script that ${testJson.setup_script_description}.`
+    : testJson.setup_script_description;
+  // As a hack, special case mode instructions for VoiceOver for macOS until we
+  // support modeless tests. ToDo: remove this when resolving issue #194
+  const modePhrase = at.name === "VoiceOver for macOS" ? "Describe " : `With ${at.name} in ${mode} mode, describe `;
 
-  if (document.readyState !== 'complete') {
-    window.setTimeout(() => {
-      displayTestPageAndInstructions(testPage);
-    }, 100);
-    return;
+  const commands = testJson.commands;
+  const assertions = testJson.output_assertions.map((a) => a[1]);
+  const additionalAssertions = testJson.additional_assertions;
+
+  const bind =
+    (fn, ...args) =>
+    (...moreArgs) =>
+      fn(...args, ...moreArgs);
+
+  return {
+    errors: {
+      visible: false,
+      header: "",
+      errors: [],
+    },
+    instructions: {
+      header: {
+        header: `Testing task: ${document.title}`,
+        focus: resultState.currentUserAction === "loadPage",
+      },
+      description: `${modePhrase} how ${at.name} behaves when performing task "${lastInstruction}"`,
+      instructions: {
+        header: "Test instructions",
+        instructions: [
+          [
+            `Restore default settings for ${at.name}. For help, read `,
+            {
+              href: "https://github.com/w3c/aria-at/wiki/Configuring-Screen-Readers-for-Testing",
+              description: "Configuring Screen Readers for Testing",
+            },
+            `.`,
+          ],
+          `Activate the "Open test page" button below, which opens the example to test in a new window${setupScriptDescription}`,
+        ],
+        boldInstructions: [modeInstructions, ...userInstructions],
```

Comment:
I think strongInstructions or emphasizedInstructions would be a better term given the fact this is typically used by AT users.

Reply:
👍